core/src/main/java/io/jenkins/pluginhealth/scoring/probes/Probe.java

```diff
@@ -34,7 +34,7 @@ public abstract class Probe {
 
     /**
      * Starts the analyze on a plugin.
-     * Should only be called by the {@link ProbeEngine#run()} method.
+     * The implementation of the probe action is deferred to {@link Probe#doApply(Plugin, ProbeContext)}.
      *
      * @param plugin  the plugin on which to perform the analyze
```

Comment:
```suggestion
     * @param plugin  the plugin on which to perform the analysis
```

Also, needs an update of the word "analysis" at 1 line below at line 41.